packages/manager/apps/catalog-revamp/src/components/Filters/Filters.scss

```diff
@@ -0,0 +1,54 @@
+.filters-container {
+  position: relative;
+  overflow: visible;
+  display: flex;
+  margin: 3em;
```

Comment:
Avoid mixing `em` and `px` it will make it harder to refactor with tailwind

Reply:
Done